### PR TITLE
Update deploy-intro.html

### PR DESCRIPTION
**Pull request by satyampsoni**
Before:
It was not clear in the mind of reader what actually are deployment configuration and instances.

changes:
Proposed the definition of Deployment configuration and instances

After: 
It gives a understanding of what deployment configuration and instances are right in the documentation.

### CODE REVIEW

**Review by Pradumnasaraf — COMMENTED** [review]
Hey,

Please sign the CLA first https://github.com/kubernetes/website/pull/40033#issuecomment-1470724536

**Comment by satyampsoni** [comment]
While signing up for CLA they are asking for an organisation to select. I don't work anywhere. I am just a university student.

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by T-Lakshmi** [comment]
Hey @satyampsoni,
The [Cloud Native Computing Foundation](https://www.cncf.io/) (CNCF) defines the legal status of the contributed code in two different types of Contributor License Agreements (CLAs), [individual contributors](https://github.com/cncf/cla/blob/master/individual-cla.pdf) and [corporations](https://github.com/cncf/cla/blob/master/corporate-cla.pdf).

> While signing up for CLA they are asking for an organisation to select. I don't work anywhere. I am just a university student.

In your case, you can choose the option `individual contributors`.
For detail information, please go through [this reference link](https://github.com/kubernetes/community/blob/master/CLA.md).  
Thanks,

**Comment by satyampsoni** [comment]
Pls review it now. I have signed the CLA

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -27,8 +27,8 @@ <h3>Objectives</h3>
                 <h3>Kubernetes Deployments</h3>
                 <p>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
-                To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
-                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
+                To do so, you create a Kubernetes <b>Deployment</b> configuration (Kubenetes Object that describes the desired state of deployment). The Deployment instructs Kubernetes
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:30]: To do so, you create a Kubernetes <b>Deployment</b> (a Kubernetes object that declares your intent to
run a set of application instances). The Deployment instructs Kubernetes

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 2]
```diff
@@ -27,8 +27,8 @@ <h3>Objectives</h3>
...
+                how to create and update instances (Copy of a Particular object) of your application. Once you've created a Deployment, the Kubernetes
                 control plane schedules the application instances included in that Deployment to run on individual Nodes in the
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:32]:
```suggestion
                how to create and update instances (Copy of a Particular object) of your application. Once you've created a Deployment, the Kubernetes
                control plane schedules those application instances associated with the that Deployment to run 
                (as containers within Pods) on individual Nodes in your
```

**Review by sftim — COMMENTED** [review]
We'll need to tweak this to make sure it's not misleading the reader.
The app instances are _not_ copies of the Deployment.

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html (hunk 1 above)
Comment (satyampsoni) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:30]: I am unable to add this suggestion to the batch. pls look into it.

**Pull request closed without merging**